slick-testkit/src/main/scala/com/typesafe/slick/testkit/tests/PlainSQLTest.scala

```diff
@@ -21,10 +21,10 @@ class PlainSQLTest extends AsyncTest[JdbcTestDB] {
     val s2 = sql"select id from USERS where name = '#${"guest"}'".as[Int]
     val s3 = sql"select id from USERS where name = $foo".as[Int]
     val s4 = sql"select id from USERS where name = '#$foo'".as[Int]
-    s1.statements.head shouldBe "select id from USERS where name = ?"
-    s2.statements.head shouldBe "select id from USERS where name = 'guest'"
-    s3.statements.head shouldBe "select id from USERS where name = ?"
-    s4.statements.head shouldBe "select id from USERS where name = 'foo'"
+    s1.statements.head shouldBe "select id from USERS where name = ? "
+    s2.statements.head shouldBe "select id from USERS where name = 'guest' "
+    s3.statements.head shouldBe "select id from USERS where name = ? "
+    s4.statements.head shouldBe "select id from USERS where name = 'foo' "
```

Comment:
where do these spaces come from? I know no trim means they are not removed, but where do they come from?